inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/provider/MongoProvider.java

```diff
@@ -33,6 +35,8 @@
  */
 public class MongoProvider implements ExtractNodeProvider {
```

Comment:
Can this class name change to `MongoDBProvider`?

Reply:
> Can this class name change to `MongoDBProvider`?

okk